fineract-provider/src/main/java/org/apache/fineract/template/service/TemplateMergeService.java

```diff
@@ -162,7 +150,7 @@ private static String getStringFromInputStream(final InputStream is) {
         String line;
         try {
 
-            br = new BufferedReader(new InputStreamReader(is));
+            br = new BufferedReader(new InputStreamReader(is, Charset.defaultCharset()));
```

Comment:
again IMHO just hard coding UTF-8 here is safer (as more stable and reproducible) than relying on platform default encoding